internal/bot.go

```diff
@@ -26,9 +28,22 @@ func NewBot(token string) (Bot, error) {
 
 	log.Printf("Authorized on account %s", botAPI.Self.UserName)
 
+	loc, err := time.LoadLocation(timezone)
+	if err != nil {
+		return bot, err
+	}
+
+	parsedTime, err := time.ParseInLocation("02 Jan 06 15:04", time.Now().Format("02 Jan 06 ")+t, loc)
```

Comment:
I am confused on what is happening here. What is the time we are parsing and why not use the time from the input?

Reply:
the time from the input is in format `hh:mm`, but here we try to parse date, so I add a formatted date `time.Now().Format("02 Jan 06 ")` to the input time `t` and parse them with location from the timezone